src/utils/token-standard/v2/holdings.ts

```
@@ -0,0 +1,550 @@
+  if (!holders.every(isNonEmptyString)) return undefined;
+
+  const expiresAt = readNullableString(value['expiresAt']);
+  const context = readNullableString(value['context']);
+  const rawExpiresAfter = value['expiresAfter'];
+  let expiresAfter: TokenStandardV2Lock['expiresAfter'];
+  if (rawExpiresAfter === null) {
+    expiresAfter = null;
+  } else if (
+    isRecord(rawExpiresAfter) &&
+    typeof rawExpiresAfter['microseconds'] === 'string' &&
+    /^-?\d+$/.test(rawExpiresAfter['microseconds'])
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)